functional_tests/scylla_operator/conftest.py

```diff
@@ -40,6 +40,7 @@
 #
 #       Which appears when we define and use fixtures in one single module like we do here.
 
+# TODO: add support for multiDC setups
```

Comment:
maybe we should write it down as task, but I think it might be a bit complex to setup 2 kind setups, or doing that only for EKS/GKE, with is also a bit complex, and would need it's own specific setup, anyhow that's something we should discuss in some task, the comment here won't trigger any action from anyone.

Reply:
It is not that complex.
The `kind` tool allows to create more than 1 K8S cluster on the local machine.
So, this task will be simpler than the multiDC support as a whole feature - for sure.

And, multiDC support for functional tests is easy to test already with EKS multiDC setup which is already tested for some extent.